docs/messaging/nats-integration.md

```diff
@@ -2,110 +2,201 @@
 title: .NET Aspire NATS integration
 description: Learn how to use the .NET Aspire NATS integration to send logs and traces to a NATS Server.
 ms.topic: how-to
```

Comment:
All of these integration articles read like conceptual or overview mixes. Maybe you should change the topic type to conceptual?

Reply:
Yes, will do. I like that idea.